corehq/apps/app_manager/dbaccessors.py

```diff
@@ -157,6 +157,25 @@ def get_built_app_ids(domain):
     return [app_id for app_id in app_ids if app_id]
 
 
+def get_built_app_ids_for_app_id(domain, app_id, version=None):
+    """
+    Returns all the built apps for an application id. If version is specified returns all apps after that
+    version.
+    """
+    from .models import Application
+    key = [domain, app_id]
+    skip = 1 if version else 0
```

Comment:
The function of `skip` here is to skip over the version specified in `version` so that its not included in the results.